src/components/Header/components/HeaderSearchForm.test.tsx

```diff
@@ -33,6 +46,8 @@ describe("HeaderSearchForm", () => {
     (window as any).location = { assign: jest.fn() };
     // Override the `Date` class so we always get the same
     // timestamp when `getTime` is called.
+    // OW: This mock is not currently not working, which
+    // is why the tests are being skipped.
```

Comment:
Gotcha! The `jest.useFakeTimers()`, `jest.runOnlyPendingTimers()`, and `jest.useRealTimers()` seem to override or not take this mock into account. Try removing those function calls and remove the skips.

Reply:
omg it works. thank you!!!! I kept jest.runOnlyPendingTimers() and jest.useRealTimers() but put them in an afterall.